forge-gui/res/cardsfolder/upcoming/invert_polarity.txt

```diff
@@ -0,0 +1,8 @@
+Name:Invert Polarity
+ManaCost:U U R
+Types:Instant
+A:SP$ FlipACoin | TargetType$ Spell | TgtZone$ Stack | WinSubAbility$ GainControl | LoseSubAbility$ CounterIt | ValidTgts$ Card | StackDescription$ SpellDescription | SpellDescription$ Choose target spell, then flip a coin. If you win the flip, gain control of that spell and you may choose new targets for it. If you lose the flip, counter that spell.
+SVar:GainControl:DB$ ControlSpell | Defined$ Targeted | Mode$ Gain | SubAbility$ DBChooseTargets
+SVar:DBChooseTargets:DB$ ChangeTargets | Defined$ Targeted | Optional$ True
+SVar:CounterIt:DB$ Counter | Defined$ Targeted
```

Comment:
None of these need `StackDescription$ None` ?

Reply:
Don't see a difference between the stackdescriptions with this ? Though I don't understand the Forge stack builder as well as you do

Reply:
@Northmoc
they probably didn't get picked up because they were only additional sub?

Reply:
I think you can just revert it so we can merge